nslsii/ad33.py

```diff
@@ -89,7 +89,7 @@ class StatsPluginV33(PluginBase):
     Due to https://github.com/areaDetector/ADCore/pull/333
     """
     _default_suffix = 'Stats1:'
-    _suffix_re = 'Stats\d:'
+    _suffix_re = 'Stats\\d:'
```

Comment:
I'm not sure about this change. Should the `\` be escaped?

Reply:
They all seem equivalent:
```py
In [1]: _suffix_re = 'Stats\d:'

In [2]: _suffix_re
Out[2]: 'Stats\\d:'

In [3]: _suffix_re2 = 'Stats\\d:'

In [4]: _suffix_re2
Out[4]: 'Stats\\d:'

In [5]: _suffix_re3 = r'Stats\d:'

In [6]: _suffix_re3
Out[6]: 'Stats\\d:'
```
but to make flake8 happy, either 2nd or 3rd option is fine. I'd rather go with the `r'...'` to match the style of recent changes in bluesky.